### PR TITLE
Programmatic layout

### PR DESCRIPTION
**Pull request by rilury**
- Added UI layout programmatically 
- Removed replaced storyboards files
- Added "No logins found" string as title for empty credential list
- Removed unused files

*The UI design remains the same.

### CODE REVIEW

File: CredentialProvider/CredentialWelcomeViewController.swift [hunk 1]
```diff
@@ -5,22 +5,73 @@
...
     }
+    
+    func addSubviews() {
+        view.addSubview(logoImageView)
+        view.addSubview(taglineLabel)
+        view.addSubview(activityIndicator)
+    }
+    
+    func addViewConstraints() {
+        view.addSubview(logoImageView)
```
Comment (razvanlitianu) [inline CredentialProvider/CredentialWelcomeViewController.swift:61]: Looks like `logoImageView` is added twice.

File: CredentialProvider/Cells/NoSearchResultCell.swift [hunk 2]
```diff
@@ -7,14 +7,48 @@ import UIKit
...
 class NoSearchResultCell: UITableViewCell {
     
-    @IBOutlet weak var titleLabel: UILabel! {
-        didSet {
-            titleLabel.text = .LoginsListNoMatchingResultTitle
+    static let identifier = "noSearchResultCell"
+    
+    lazy private var titleLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = UIColor(named: "labelColor")
```
Comment (razvanlitianu) [inline CredentialProvider/Cells/NoSearchResultCell.swift:14]: Can we extract colors like this into constants? Do we have them already?

File: CredentialProvider/CredentialWelcomeViewController.swift (hunk 1 above)
Comment (rilury) [reply CredentialProvider/CredentialWelcomeViewController.swift:61]: It was a mistake, I will remove the duplicate. Thank you

File: CredentialProvider/Cells/NoSearchResultCell.swift (hunk 2 above)
Comment (rilury) [reply CredentialProvider/Cells/NoSearchResultCell.swift:14]: Ok, I will check and if we do not have them, I will extract them into constants, as you suggested. Thank you

**Review by st3fan — APPROVED** [review]
Thank you this looks great. Let's merge it in the `credentialprovider` branch and then we can ask @boek for a final review.

**Pull request merged**